web/src/components/eventslist-component/EventFilters.js

```diff
@@ -0,0 +1,77 @@
+import React, {Component} from 'react';
+import Cards from './EventCards';
+import {
+  ToggleButton,
+  ToggleButtonGroup,
+  Jumbotron
+} from 'react-bootstrap';
+
+
+class EventFilters extends Component {
+  constructor(props) {
+    // get area selection from props
+    super(props);
+
+    // save the arrays of filter names and filter toggle(selected or not) in state
```

Comment:
Can you go over all comments and make sure first letter of the first word is capitalized and there's a . at the end of the sentence?